qa-pipelines/README.md

```diff
@@ -109,3 +110,21 @@ The composability of the pipeline tasks means there are some interesting things
 In order to do this, replace the CAP bundle URLs in the pipeline config, enable *only* the pre-upgrade tests, and deploy the pipeline. The deploy will use the bundle from the `cap-sle-url` or `cap-opensuse-url` settings in the pipeline config.
 ## Continue a test suite from where a previous build left off.
 Sometimes running tests may fail for timing-related reasons which may be intermittent. If this happens, and you want to try to re-run the test and continue the build from where it left if, you can deploy a new pipeline with only the failed test and following tasks enabled, unlock the config which was used, and run a build from the new pipeline
+
+
+# Dev Nightly Upgrades CI
+
+Nightly Builds are builds which happen every night from develop branch of scf. The build lands in s3://cap-release-archives/nightly/
+The idea here is to test bare minimum of these nightly builds, i.e.
+1. We want to make sure `helm install` and `helm upgrades` are not broken due to any changes
+2. Also, catch well in advance, if any changes to scf have broken qa-pipelines
+
+Dev ECP pool: https://github.com/SUSE/cf-ci-pools/tree/cap-ci-kube-hosts
+
+Concourse config for Dev ECP pool: [config-capci-nightly.yml](config-capci-nightly.yml)
+
+Example command to deploy CI on concourse:
+
+`./set-pipeline -t provo -p Official-DEV-Nightly-Upgrades --pool=capci --nightly pipeline-presets/cap-qa-upgrades-lite.yml`
+
+[cap-qa-upgrades-lite.yml](pipeline-presets/cap-qa-upgrades-lite.yml) is more than enough to accomplish our goals here
```

Comment:
I'd recommend changing this to [pipeline-presets/cap-qa-upgrades-lite.yml](pipeline-presets/cap-qa-upgrades-lite.yml) for clarity